docs/source/contributing_examples/test_example.py

```diff
@@ -8,7 +8,7 @@
 def test_track():
     default_trackid = "some_id"
     data_home = "tests/resources/mir_datasets/dataset"
-    dataset = example.Dataset(data_home)
+    dataset = example.Dataset(data_home, version="test")
```

Comment:
I'm not sure I follow why this is needed here...

Reply:
For the tests, it uses the "test" index, instead of the "default" one. Right now, in all cases except acousticbrainz, the test and default versions are the same. But for acousticbrainz, the test version is the small local index, whereas the default one is more than 1 GB and remote.

Reply:
Gotcha, so you want developers to test the remote indexes in a transparent way. Right now tests for `acousticbrainz` run sneakily using the small index but is not transparent, am I getting this right?